documentation/book/snip-external-logging-Kafka.adoc

```diff
@@ -0,0 +1,37 @@
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: {kind}
```

Comment:
There's no point in using the attribute `{kind}` here, since this is necessarily the `Kafka` kind.
```suggestion
kind: Kafka
```